filters/meson.build

```diff
@@ -0,0 +1,354 @@
+    'average',
+    'backproject',
+    'bin',
+    'binarize',
+    'blur',
+    'buffer',
+    'calculate',
+    'clip',
+    'crop',
+    'cut',
+    'cut-sinogram',
+    'center-of-rotation',
+    'concatenate-result',
+    'contrast',
+    'correlate-stacks',
+    'dfi-sinc',
+    'denoise',
+    'detect-edge',
+    'rescale',
+    'dummy-data',
+    'dump-ring',
+    'duplicate',
+    'filter',
+    'flatten',
+    'flatten-inplace',
+    'flat-field-correct',
+    'fftmult',
+    'filter-particle',
+    'filter-stripes',
+    'filter-stripes1d',
+    'flip',
+    'forwardproject',
+    'get-dup-circ',
+    'gradient',
+    'horizontal-interpolate',
+    'interpolate',
+    'interpolate-stream',
+    'loop',
+    'map-slice',
+    'map-color',
+    'mask',
+    'measure',
+    'measure-sharpness',
+    'median-filter',
+    'memory-in',
+    'memory-out',
+    'merge',
+    'metaballs',
+    'monitor',
+    'null',
+    'opencl',
+    'opencl-reduce',
+    'ordfilt',
+    'pad',
+    'polar-coordinates',
+    'power-spectrum',
+    'reduce',
+    'refeed',
+    'remove-circle',
+    'ring-pattern',
+    'ringwriter',
+    'replicate',
+    'remove-outliers',
+    'rotate',
+    'segment',
+    'sleep',
+    'slice',
+    'sliding-stack',
+    'stack',
+    'stacked-backproject',
+    'stitch',
+    'stdin',
+    'tile',
+    'transpose',
+    'transpose-projections',
+    'swap-quadrants',
+    'subtract',
+    'unsplit',
+    'volume-render',
+    'zeropad',
+]
+
+fft_plugins = [
+    'fft',
+    'ifft',
+    'retrieve-phase',
+]
+
+zmq_plugins = [
+    'zmq-pub',
+    'zmq-sub',
+]
+
+kernels = [
+]
+
+read_sources = [
+    'ufo-read-task.c',
+    'readers/ufo-reader.c',
+    'readers/ufo-edf-reader.c',
+    'readers/ufo-raw-reader.c',
+]
+
+write_sources = [
+    'ufo-write-task.c',
+    'writers/ufo-writer.c',
+    'writers/ufo-raw-writer.c',
+]
+
+tiff_dep = dependency('libtiff-4', required: false)
+hdf5_dep = dependency('hdf5', required: false)
+jpeg_dep = dependency('libjpeg', required: false)
+pangocairo_dep = dependency('pangocairo', required: false)
+opencv_dep = dependency('opencv', required: false)
+uca_dep = dependency('libuca', required: false)
+clblast_dep = dependency('clblast', required: false)
+clfft_dep = dependency('clFFT', required: false)
+zmq_dep = dependency('libzmq', required: false)
+json_dep = dependency('json-glib-1.0', version: '>=1.1.0', required: false)
+
+conf = configuration_data()
+conf.set('HAVE_AMD', clfft_dep.found())
+conf.set('HAVE_TIFF', tiff_dep.found())
+conf.set('HAVE_JPEG', jpeg_dep.found())
+conf.set('WITH_HDF5', hdf5_dep.found())
+conf.set('BURST', get_option('lamino_backproject_burst_mode'))
+conf.set('CL_TARGET_OPENCL_VERSION', '120')
+
+configure_file(
+    input: 'config.h.meson.in',
+    output: 'config.h',
+    configuration: conf,
+)
+
+# standard plugins
+
+foreach plugin: plugins
+    shared_module(plugin,
+        'ufo-@0@-task.c'.format(plugin),
+        dependencies: deps,
```

Comment:
deps does not seem to be defined so it will use the value from ufo-core, which has the wrong dependencies
Should have something like
```
deps = [
    lib_dep,  // from ufo-core/ufo/meson.build
    opencl_dep,
    m_dep,
]
```